tests/unittests/Test-origin.cpp

```diff
@@ -24,13 +28,58 @@ const std::vector<double> example_point_C2HDM{/* lambda_1 = */ 3.29771,
                                               /* Re(m_{12}^2) = */ 2706.86,
                                               /* tan(beta) = */ 4.64487,
                                               /* Yukawa Type = */ 1};
+
+} // namespace
+
+/**
+ * @test Check if the automatic Debye corrections match the SM one in the SM
+ * case. This should be y_t^2/4.
+ */
+TEST_CASE("Test Calculate Debye", "[origin-s]")
```

Comment:
Why are you using the new tag `[origin-s]` for the Debye-test? What does it stand for?

Reply:
Fixed it. It was useful during debugging